internal/service/account/resource_account_authentication.go

```diff
@@ -16,22 +16,32 @@ var aivenAccountAuthenticationSchema = map[string]*schema.Schema{
 		Required:    true,
 		Description: "The unique id of the account.",
 	},
+	"enabled": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: schemautil.Complex("Status of account authentication method.").DefaultValue(false).Build(),
+	},
+	"authentication_id": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Account authentication id",
+	},
+	"name": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "The name of the account authentication.",
+	},
 	"type": {
 		Type:         schema.TypeString,
 		Required:     true,
 		ValidateFunc: validation.StringInSlice([]string{"internal", "saml"}, false),
 		Description:  schemautil.Complex("The account authentication type.").PossibleValues("internal", "saml").Build(),
 	},
-	"name": {
```

Comment:
we usually arrange fields in this order: 
- first required fields
- then optional
- then computed